### DISCUSSION TITLE
entry specific browser integration

### DISCUSSION THREAD

**Discussion by rfoppen** [keepassxreboot/keepassxc#6098] [Q&A] [answered]
I have keepassxc connected to multiple browsers.  

Can I configure an entry so that it is available for use and selection / autotype in e.g. FireFox but not in Chrome or vice versa

Sort of similar to Override URL

**Comment by droidmonkey** [comment, marked as answer]
No we don't do per browser allowance.

**Reply by rfoppen** [reply]
short and sweet I suppose. 

Would be nice to have :)
